sdk/monitor/azure-monitor-opentelemetry/setup.py

```diff
@@ -83,17 +83,17 @@
     install_requires=[
         "azure-core<2.0.0,>=1.28.0",
         "azure-core-tracing-opentelemetry~=1.0.0b11",
-        "azure-monitor-opentelemetry-exporter~=1.0.0b31",
-        # TODO: Unpin once breaking change in 1.32.0/0.53b0 is resolved.
-        "opentelemetry-instrumentation-django<0.53b0,>=0.49b0",
-        "opentelemetry-instrumentation-fastapi<0.53b0,>=0.49b0",
-        "opentelemetry-instrumentation-flask<0.53b0,>=0.49b0",
-        "opentelemetry-instrumentation-psycopg2<0.53b0,>=0.49b0",
-        "opentelemetry-instrumentation-requests<0.53b0,>=0.49b0",
-        "opentelemetry-instrumentation-urllib<0.53b0,>=0.49b0",
-        "opentelemetry-instrumentation-urllib3<0.53b0,>=0.49b0",
+        "azure-monitor-opentelemetry-exporter~=1.0.0b36",
+        "opentelemetry-instrumentation-django~=0.54b1",
+        "opentelemetry-instrumentation-fastapi~=0.54b1",
+        "opentelemetry-instrumentation-flask~=0.54b1",
+        "opentelemetry-instrumentation-psycopg2~=0.54b1",
+        "opentelemetry-instrumentation-requests~=0.54b1",
+        "opentelemetry-instrumentation-urllib~=0.54b1",
+        "opentelemetry-instrumentation-urllib3~=0.54b1",
         "opentelemetry-resource-detector-azure~=0.1.4",
-        "opentelemetry-sdk<1.32,>=1.28.0",
+        "opentelemetry-api<2.0,>=1.33.1",
+        "opentelemetry-sdk<2.0,>=1.33.1",
```

Comment:
Is there a reason for raising the minimum? For attach, I currently need the older version until the breaking change is fixed.

Reply:
reverted